src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LeaderBoard app</title>
+</head>
+<body>
+    <header><h1>Leaderboard</h1></header>
+    <main>
+        <section class="leaderboard">
+            <div class="heading">
+                <h2 class="head">Recent scores</h2>
+                <button id="refresh" class="btn" type="button">Refresh</button>
+            </div>
+            <div class="score-list">
+                <p class="rank">Name : 100</p>
+                <p class="rank">Name : 134</p>
+                <p class="rank">Name : 20</p>
+                <p class="rank">Name : 60</p>
+                <p class="rank">Name : 10</p>
+                <p class="rank">Name : 200</p>
+                <p class="rank">Name : 199</p>
+                <p class="rank">Name : 150</p>
+            </div>
+        </section>
+        <section class="form">
+            <h2 class="head">Add your score</h2>
+            <form >
+                <input type="text" class="form-control" id="name" placeholder="Your name">
+                <input type="text" class="form-control" id="score" placeholder="Your score">
+                <input type="submit" value="Submit" id="submit" class="btn">
+            </form>
+        </section>
```

Comment:
- [ OPTIONAL ] Kindly take care of the HTML & CSS best practices :point_down: and add two spaces of indentation instead of four

![1111](https://user-images.githubusercontent.com/57562869/152203471-6c7a42da-2a1f-4f34-b92e-67d5d3cc3923.png)